components/Header/index.js

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import Button from "../Buttons/Button";
+
+const Header = ({ pageHeader }) => {
+  const { backgroundColour, imgSrc, imgAlt, title, caption } = pageHeader;
+  return (
+    <header>
+      <div className="flex justify-between">
+        <div className="self-center mx-16 w-1/2">
+          <h2 className="py-7 font-bold text-8xl">{title}</h2>
```

Comment:
Do you think it would be worth adding some of those responsive classes to the text? To make it a little smaller on smaller screens?